### PR TITLE
build: adds new script to clean package lock files

### PR DESCRIPTION
**Pull request by awentzel**
<body>
Adds a build utility script in the case that errors occur because package lock files are a mess.
<footer>

### CODE REVIEW

File: build/clean-locks.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/**
+ * Utility for cleaning package locks.
+ * Usage: node build/clean-locks.js
```
Comment (nicholasrice) [inline build/clean-locks.js]: how would you feel about changing this to "clean-package-locks"? That would communicate more accurately what this is doing - I might not know at a glace what a "lock" is nor why I might clean it.

File: build/clean-locks.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+var rimraf = require('rimraf');
+
+glob('./packages/**/package-lock.json', (err, files) => {
+    if (err) {
+        throw err;
+    }
+
+    files.forEach(file => {
+        rimraf(file, (err) => {
+            console.log(file, "deleted");
```
Comment (nicholasrice) [inline build/clean-locks.js]: you should throw if `err` exists

File: build/clean-locks.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+/**
+ * Utility for cleaning package locks.
+ * Usage: node build/clean-locks.js
+ */
+
+var glob = require('glob');
+var rimraf = require('rimraf');
+
+glob('./packages/**/package-lock.json', (err, files) => {
```
Comment (nicholasrice) [inline build/clean-locks.js]: you'll need to enumerate `glob` as a dependency - or, probably a better route - `rimraf` actually supports globbing, so you may be able to omit that dependency and change this code to just `rimraf('./packages/**/package-lock.json', (err) => { ... })`

File: build/clean-package-locks.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+var rimraf = require('rimraf');
+
+rimraf('./packages/**/package-lock.json', (err) => {  
+    if (err) {
+        throw err;
+    }
+
+    console.log("All `package-lock.json` files have been deleted");
+})
```
Comment (chrisdholt) [inline build/clean-package-locks.js]: Should have a newline

**Pull request merged**